tools/update_ruby_gem_packages.rb

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+# update_ruby_gem_packages version 1.0 (for Chromebrew)
+# Author: Satadru Pramanik (satmandu) satadru at gmail dot com
+# Usage in packages dir: ../tools/update_ruby_gem_packages.rb
+Dir['ruby_*.rb'].each do |package|
+  gem_name = package.gsub('.rb', '').gsub('ruby_', '').gsub('_', '-')
+  gem_version = Gem.latest_spec_for(gem_name).version.to_s
+  puts "Updating #{gem_name} package file to #{gem_version}"
+  system "sed -i \"s,.*version.*,\ \ version '#{gem_version}-ruby-#{RUBY_VERSION.slice(/(?:.*(?=\.))/)}',\" #{package}"
+end
```

Comment:
Could this be a crew command instead of a script?  Seems like something we could use often.

Reply:
Maybe it makes sense to integrate into the `crew upgrade` process?

Reply:
Yes, in theory it could be a part of `crew update` but that would really slow that down.

This is better done in batch mode occasionally externally maybe?

Reply:
But I do think adding more to crew for that might be another PR... And you then also get sync issues with packages in the repo... So maybe we think about ignoring the versioning entirely, and only looking for version numbers during `crew update` for installed packages.

Reply:
Is this still needed since we're updating gems weekly in crew now?

Reply:
Actually, now that I think about it, maybe it makes sense to set an environment variable or constant for the update period instead of hard coding it to a week?  That way we can set the duration period to zero if we want to update right away.

Reply:
This is also what I was thinking as I went to bed... so I changed it accordingly.

Reply:
100%. If `crew update` is our proxy for version checks for our packages, we need to be automating getting version updates into our packages, especially for packages that we don't have to build ourselves, as that's low hanging fruit...